src/include/binder/bind_node_visitor.h

```diff
@@ -114,6 +117,32 @@ class BindNodeVisitor : public SqlNodeVisitor {
                                common::ErrorCode::ERRCODE_FEATURE_NOT_SUPPORTED);
     }
   }
+
+  std::vector<common::ManagedPointer<parser::AbstractExpression>> UnifyOrderByExpression(
+      common::ManagedPointer<parser::OrderByDescription> order_by_description,
+      const std::vector<common::ManagedPointer<parser::AbstractExpression>> &select_items) {
+    auto exprs = order_by_description->GetOrderByExpressions();
+    auto size = order_by_description->GetOrderByExpressionsSize();
+    for (size_t idx = 0; idx < size; idx++) {
+      if (exprs[idx].Get()->GetExpressionType() == terrier::parser::ExpressionType::VALUE_CONSTANT) {
+        auto constant_value_expression = exprs[idx].CastManagedPointerTo<parser::ConstantValueExpression>();
+        type::TypeId type = constant_value_expression->GetReturnValueType();
+        int64_t column_id = 0;
+        if (type == type::TypeId::INTEGER) {
+          column_id = constant_value_expression->GetInteger().val_;
+        } else if (type == type::TypeId::DECIMAL) {
+          column_id = constant_value_expression->GetReal().val_;
+        }
```

Comment:
What about other data types? You may want to make this a `switch` and then have a default option.

Reply:
I've changed this to switch.